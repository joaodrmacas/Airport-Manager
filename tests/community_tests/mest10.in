a GTM UnitedStates Gotham City
a DUB Ireland		Dublin
l BUD
l
q
