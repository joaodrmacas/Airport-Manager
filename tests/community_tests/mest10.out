airport GTM
airport DUB
BUD: no such airport ID
DUB Dublin Ireland 0
GTM Gotham City UnitedStates 0
